src/main/java/sql/dao/mysql/CustomerDAO.java

```diff
@@ -60,6 +59,11 @@ public void save(Customer o) {
         saveWithTryCatch(phonesql, valueList, typeList);
     }
 
+    @Override
+    public void savePhone(Customer o) {
+
+    }
+
```

Comment:
The method was inherited by the interface. I added this method so it will work as intended for MyBatis. In this case, the way I implemented the MySQL, makes this method unnecessary.